### DISCUSSION TITLE
Version v3.9.1 prone to DNS leaks?

### DISCUSSION THREAD

**Discussion by MervisDiamonds** [bubuntux/nordvpn#161] [Q&A]
I've been v3.7.4 for a while, and I was curious if it was worth upgrading to v3.9.1. 

The comment on v3.8.10 says it could have DNS leaks. Does the new version have the same risk?

**Comment by bubuntux** [comment]
It could be some dns leak at bootstrap, i would recommend override the dns for the container.

**Reply by bubuntux** [reply]
also worth to mention that all versions could have this leak at bootstrap if you kernel host does not support group filtering for the iptables (very common on NAS)
